docs/prover/changelog.md

```diff
@@ -4,6 +4,37 @@ Release Notes
 ```{contents}
 ```
 
+4.10.1 (August 21, 2023)
+------------------------
+
+### CVL
+- feature: Support Solidity calls also from internal summaries
```

Comment:
I think this release introduces calls from both internal and external summaries right?  If so, it should probably just say "Support Solidity calls from summaries" or maybe "summaries can now call contract functions"

Reply:
I think 4.8.0 supported it in external functions